scripts/ci/bump-version

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+[ -n "${DEBUG}" ] && set -x
+
+# This file is intended to run on Jenkins, if you want to create a
+# release locally, just run
+# 1. npm run release
+# followed by
+# 2. git push --follow-tags ${BRANCH_NAME}
+
+## Configuration
+#####################################################################
+
+# really create a release? (prevents accedential calling)
+CREATE_RELEASE=${CREATE_RELEASE:-0}
+
+# really push release? (prevents accedential calling)
+PUSH_RELEASE=${PUSH_RELEASE:-0}
+
+# get release target `master`` is fine, but if BRANCH_NAME is
+# `release/1.11` we only want `1.11`
+PKG_TARGET=$(echo ${BRANCH_NAME} | sed -E 's/([^/]*)$|./\1/g')
```

Comment:
💡 I'd use a file that points the respective version instead of _parsing_ the name from the branch name.